sktime/highlevel.py

```diff
@@ -127,21 +126,26 @@ def pred_horizon(self):
 class BaseStrategy:
     """A meta-estimator that employs a low level estimator to
     perform a pescribed task
-
     Parameters
     ----------
     estimator : BaseEstimator
         An instance of an appropriately initialized
         low-level estimator
     """
-    def __init__(self, estimator):
+    def __init__(self, estimator, *args, **kwargs):
```

Comment:
I would make it an explicit kwarg with defaults. 
```suggestion
    def __init__(self, estimator, name=None):
```

Reply:
ok, *args, **kwargs was in the original code. I assumed this was needed for future extensions.